apollo-portal/src/main/resources/static/app/cluster_manage.html

```diff
@@ -0,0 +1,146 @@
+<!--
+  ~ Copyright 2024 Apollo Authors
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+<!doctype html>
+<html ng-app="cluster_manage">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <link rel="icon" href="../img/config.png">
+  <!-- styles -->
+  <link rel="stylesheet" type="text/css" href="../vendor/bootstrap/css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="../vendor/angular/angular-toastr-1.4.1.min.css">
+  <link rel="stylesheet" type="text/css" media='all' href="../vendor/angular/loading-bar.min.css">
+  <link rel="stylesheet" type="text/css" href="../styles/common-style.css">
+  <link rel="stylesheet" type="text/css" href="../vendor/select2/select2.min.css">
+  <title>{{'Config.ClusterManage' | translate }}</title>
+</head>
+
+<body>
+
+<apollonav></apollonav>
+
+<div class="container-fluid apollo-container project-setting" ng-controller="ClusterManageController">
+  <section class="col-md-10 col-md-offset-1 panel">
+    <header class="panel-heading">
+      <div class="row">
+        <div class="col-md-9">
+          <h4 class="modal-title">{{'Config.ClusterManage' | translate }} (
+            {{'Common.AppId' | translate }}:<label ng-bind="appId"></label> )
+          </h4>
+        </div>
+        <div class="col-md-3 text-right">
+          <a type="button" class="btn btn-info" data-dismiss="modal"
+             href="{{ '/config.html' | prefixPath }}?#appid={{pageContext.appId}}">{{'Common.ReturnToIndex' | translate }}
```

Comment:
This return button should return to the project page instead of the apollo home page.
![image](https://github.com/user-attachments/assets/f2c319c8-0e89-4f6d-9bdb-9980350c9bdc)